admin-dev/themes/default/sass/partials/_rtl.sass

```diff
@@ -152,30 +152,28 @@ html
 					border-left: 0 !important
 					left: 2px
 					right: auto !important
-		a.disabled
-			.chevron:after
-				border-right: 14px solid #ccc
-				border-left: 0 !important
-		a.selected
-			.chevron:after
-				border-right: 14px solid $main-color
-				border-left: 0 !important
-		a.done
-			.chevron:after
-				border-right: 14px solid $brand-success
-				border-left: 0 !important
-	.msgBox
-		float: right !important
-		margin: 4px 5px 0 0 !important
-		.content
+			&.disabled
+				.chevron:after
+					border-right: 14px solid #ccc
+					border-left: 0 !important
+			&.selected
+				.chevron:after
+					border-right: 14px solid $main-color
+					border-left: 0 !important
+			&.done
+				.chevron:after
+					border-right: 14px solid $brand-success
+					border-left: 0 !important
+	.actionBar
+		.msgBox
 			float: right !important
-	#carrier_logo_block
-		left: 10px
-		right: auto !important
-	.actionBar a
```

Comment:
carrier_logo_block was my mistake (no need for that).
the actionbar moved above (better)